doc/multi-run.mzQC.md

```diff
@@ -0,0 +1,310 @@
+```
+          "inputFiles": [
+            {
+              "name": "techRep1_healthy",
+              "location": "c:\msdata\techRep1_healthy.raw",
+              ...
+            },
+            {
+              "name": "techRep2_healthy",
+              "location": "c:\msdata\techRep2_healthy.raw",
+              ...
+            },
+            {
+              "name": "techRep3_healthy",
+              "location": "c:\msdata\techRep3_healthy.raw",
+              ...
+            }
+          ]
+```
+The `inputFile` object is only sketched here. It can contain a lot more information, such as file format and further properties. See the full example below or `single-run.mzQC` for details.
+
+In  `qualityMetrics`, we will store the actual QC information for a particular `setQuality`. Each `qualityMetric` has an `accession` and the corresponding `name` are defined by the QC CV (see `qc-cv.obo`) and should be represented exactly as stated in the .obo file. The `value` carries the actual information.
+Metric `value`s can be either single values, tuple of values, or matrices or tables (shown in other examples). 
+
+Here we show two metrics: the first describes the relative total intensity of all contaminant proteins (from all runs in the set) -- the higher the value the more contaminants are present in the samples of the set.
+```
+            "accession": "QC:0000000",
+            "name": "Protein contaminant intensity ratio",
+            "value": 0.25
+```
+ ... and the second metric is a PCA result, which stores the first 5 principal components (PC) for this set
+```
+            "accession": "QC:0000000",
+            "name": "PrincipalComponents5_RawProteinGroupIntensity",
+            "value": [ -43.22, 32.1, 3.8, -7.7, 140.6]
+```
+As you might have noticed, the PCA metric is a bit special, because the values are meaningless unless we know the PC's for other sets to puzzle together the PCA result matrix (this is how the PCA was done in the first place: from a table, where columns represent one or more Raw files (a set), and each row is a proteingroup's intensity; you would find this data, for example, in MaxQuant's proteinGroups.txt). To make sure we can identify all sets which belong to this 'superset' for our PCA result, we annotate the `metadata` section with a CV term. All sets which have the same term (and value!), contribute one row of our PCA result matrix.
+```
+  "metadata" : {
+    ...,
+    "cvParameters": [
+      {
+        "accession": "QC:0000000",
+        "name": "superset-label",
+        "value": "MaxQuant experiment groups"
+      }
+    ]
+  }
+```
+In other words: if you need to represent some kind of hierarchy of QC information, maybe due to your experimental design, `setQuality`'s are always the leaf nodes in that tree. To describe data for inner nodes, annotate the sets in the leaf nodes with a CV term. This allows to reconstruct this hierarchy.
```

Comment:
Maybe we could build up one-by-one, first explaining set and label, then super-sets and other metadata cv parameters?